config/config.go

```diff
@@ -13,26 +13,24 @@ import (
 
 type (
 	SystemConfig struct {
-		LogType                 string         `yaml:"LogType"`
-		GNUNetcatPath           string         `yaml:"GNUNetcatPath"`
-		BaseInstallDirectory    string         `yaml:"BaseInstallDirectory"`
-		BatchSize               int            `yaml:"BatchSize"`
-		DB                      string         `yaml:"Database"`
-		HostIntelDB             string         `yaml:"HostIntelDB"`
-		ExternalHostsCollection string         `yaml:"ExternalHostsCollection"`
-		DatabaseHost            string         `yaml:"DatabaseHost"`
-		LogLevel                int            `yaml:"LogLevel"`
-		Prefetch                float64        `yaml:"Prefetch"`
-		Whitelist               []string       `yaml:"WhiteList"`
-		BlacklistedConfig       BlacklistedCfg `yaml:"BlackListed"`
-		DnsConfig               DnsCfg         `yaml:"Dns"`
-		DurationConfig          DurationCfg    `yaml:"Duration"`
```

Comment:
The duration module was removed a while ago, but config wasn't cleaned. Also we never use the BaseInstallDirectory field.... External hosts collection is currently hard coded I believe... So I took it out of the config until someone implements changing the name.